src/SourceBuild/content/build.sh

```diff
@@ -128,6 +129,13 @@ while [[ $# > 0 ]]; do
     -test|-t)
       test=true
       ;;
+    -sign)
+      properties+=( "/p:Sign=true" )
+      # Force dry run signing for now. In typical VMR builds, the official build ID is set for each repo, which
+      # tells the signing infra that it should expect to see signed bits. This won't be the case in CI builds,
+      # and won't be the case for official builds until more of the real signing infra is functional.
+      properties+=( "/p:ForceDryRunSigning=true" )
```

Comment:
Is there an open issue tracking the removal this? If not, we should consider opening one & linking it here.

Reply:
https://github.com/dotnet/source-build/issues/4678